doc/nrf/external_comp/coremark.rst

```diff
@@ -0,0 +1,88 @@
+.. _integration_template:
+
+CoreMark integration
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+CoreMark is a CPU benchmark provided by EEMBC (Embedded Microprocessor Benchmark Consortium) to verify SoCs performance.
+It provides a series of test done in each benchmark iteration.
+The number of iterations DUT can perform per second gives benchmark score.
+Integrating CoreMark into |NCS| allows users to easily verify performance of acquired SoCs.
+
+
+Integration prerequisites
+*************************
+
+Before you start the |NCS| integration with CoreMark, make sure that the following prerequisites are completed:
+
+* :ref:`Installation of the nRF Connect SDK <installation>`.
+* Setup of the required :term:`Development Kit (DK)`.
+
+Solution architecture
+*********************
+
+CoreMark repository is added by default as a module to |NCS| by adding CoreMark repository to |NCS| west manifest.
+It can be found under path :file:`modules/benchmark/coremark`.
+To run CoreMark correctly SoC needs a porting layer.
+The porting layer for devices supported in |NCS| was added and placed in: :file:`nrf/modules/coremark`.
+
+CoreMark benchmark consists of continuously repeated iterations.
+Each iteration in turn consists of workloads: Linked List, Matrix Multiply, and State Machine.
+For detailed description of each of those workloads please reefer to CoreMark README: `CoreMark GitHub`.
+
+Integration steps
+*****************
+
+To integrate CoreMark into |NCS|, after completing the prerequisites, complete the following steps:
```

Comment:
```suggestion
To integrate CoreMark into the |NCS|, complete the following steps:
```
Since they're prerequisites it's probably not necessary to repeat that.
Not really sure if this description is enough overall (e.g., template refers to this as an example: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/external_comp/bt_fast_pair.html#ug-integrating-fast-pair )

Reply:
This integration is much simpler then fast pair so it doesn't have such big  Integration section